static/src/javascripts/projects/commercial/modules/dfp/prepare-googletag.js

```diff
@@ -48,6 +48,17 @@ initMessenger(
     disableRefresh
 );
 
+let hasTargettingResolve: (value?: void) => void;
+let hasTargettingPromise: Promise<void> = new Promise(resolve => {
+    hasTargettingResolve = resolve;
+});
```

Comment:
`init` will now be <pending> until `setPageTargeting` has completed.

This behaviour is assumed based on the tests, which expect `googletag.pubads().setTargeting` to have been called once initialisation is complete.